mobileapp/components/register-login/SignIn.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  View,
+  Modal,
+  KeyboardAvoidingView,
+  Platform,
+  Pressable,
+  Image,
+} from "react-native";
+import * as React from "react";
+import { TextInput, TouchableOpacity } from "react-native-gesture-handler";
+import { useSelector } from "react-redux";
+import * as Actions from "../../redux/actions";
+import * as Utility from "../../redux/utility.js";
+import * as styles from "../../constants/account";
+import { State } from "../../redux/reducer";
+import { store } from "../../redux/store";
+import { clearLoginInformation } from "../../redux/actions";
+
+const SignIn = ({ navigation }: any) => {
```

Comment:
Any, but let's say this one is ok

Reply:
I had to downgrade the react-native-gesture-handler to version 1.8.0 to make that app work on android.